lib/ico-validator/engine.rb

```diff
@@ -1,3 +1,5 @@
+require 'rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.